### ISSUE TITLE
How to distinguish the source of getSignatures and setSignatures

### ISSUE THREAD

**Issue by caochenlei** [TypeStrong/typedoc#2095]
## Question
I don't know the corresponding relationship between them
```
"sources": [
	{
		"fileName": "a.ts",
		"line": 6,
		"character": 12
	},
	{
		"fileName": "a.ts",
		"line": 12,
		"character": 12
	}
],
"getSignature": { ... },
"setSignature": { ... }
```

----------------------------------------------------------
a.ts content:
```
interface DelegateInterface<T> {
        /**
         * @description lalala
         * @effect 1.0
         */
        set useUpdate(v: boolean);
		
	/**
         * @description lalala
         * @effect 1.0
         */
        get useUpdate(): boolean;
}
```

----------------------------------------------------------
The order in the source seems to be stored from top to bottom, but the writing order of set and get is uncertain, so I cannot determine the corresponding order of set and get

**Comment by caochenlei** [comment]
```
interface DelegateInterface<T> {

        
        /**
         * @description 是否使用更新set
         * @effect 调用端生效
         */
        set useUpdate(v: boolean);
		
		
		/**
         * @description 是否使用更新get
         * @effect 调用端生效
         */
        get useUpdate(): boolean;
		

}
```

**Comment by caochenlei** [comment]
The order in the source seems to be stored from top to bottom, but the writing order of set and get is uncertain, so I cannot determine the corresponding order of set and get

**Comment by Gerrit0** [comment]
In 0.23, since sources aren't added to signature reflections, this isn't possible cleanly - I've addressed this in the 0.24 beta branch already, which I intend to release before the end of the year.

It should be stable to check `getSignature.id` and `setSignature.id`, these should be ordered in the same way as the sources... but that's really ugly. You could also use a plugin to listen to [EVENT_CREATE_SIGNATURE](https://typedoc.org/api/classes/Converter.html#EVENT_CREATE_SIGNATURE), and use that to get the source information yourself (likely also using [addSerializer](https://typedoc.org/api/classes/Serializer.html#addSerializer) to make it show up in the JSON)

**Comment by caochenlei** [comment]
thanks
